### ISSUE TITLE
Events Operation - Bug

### ISSUE THREAD

**Issue by x30r** [null-open-security-community/swachalit#64]
Once the date of the event is passed, it is not possible to update the dates of the event in case the event is postponed. (It is currently showing under old events. Update event option exists but does not actually update in the DB after form submission) {Event #761}

For the reference, please check null Ahmedabad details.

**Comment by abhisek** [comment]
Events are meant to immutable once it is finished. Again the justification for doing this is prevent tampering of past data by new leads. Please get in touch with me over email using the same email with which you are a registered chapter lead and I can do the necessary changes for you.
